src/SFA.DAS.LevyTransferMatching.Web/Controllers/ApplicationsController.cs

```diff
@@ -36,10 +39,37 @@ public async Task<IActionResult> Application(ApplicationRequest request)
             {
                 return View(viewModel);
             }
-            else
+
+            return NotFound();
+        }
+
+        [HttpPost]
+        [Route("/accounts/{encodedAccountId}/applications/{encodedApplicationId}")]
+        public async Task<IActionResult> Application([FromServices] AbstractValidator<ApplicationPostRequest> validator, ApplicationPostRequest request, CancellationToken cancellationToken = default)
+        {
+            var validationResult = await validator.ValidateAsync(request, cancellationToken);
```

Comment:
I based it on the ApplicationDetails post method in the opportunities controller. If javascript is disabled in the browser then this would be where the request gets validated, and any model state errors are pushed back to the browser so I'd have thought that it was necessary

Reply:
read the docs and it wasn't needed